src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVVariantCallerUtils.java

```diff
@@ -130,15 +130,37 @@ static int getNumSoftClippingBases(final boolean fromStart, final List<CigarElem
      * @param cigarElements
      */
     @VisibleForTesting
-    static void checkCigarElements(final List<CigarElement> cigarElements) {
+    static void validateCigar(final List<CigarElement> cigarElements) {
```

Comment:
You could update the comment for this method, which still describes the redundant check you removed.